src/mui/input/EmbeddedArrayInput.js

```diff
@@ -182,6 +180,8 @@ export class EmbeddedArrayInput extends Component {
             disabled,
             customButtons,
             actionsContainerStyle,
+            innerContainerStyle,
+            allowDivider,
```

Comment:
Name is a bit misleading, let's rename this to `insertDivider` since it insert dividers. If you have other suggestions please write them in the comments.

Reply:
I think `insertDividers` good enough. I will make change accordingly.